### DISCUSSION TITLE
Welcome to Discussions!

### DISCUSSION THREAD

**Discussion by aahnik** [aahnik/wappdriver#38] [General]
## 👋 Welcome!

# GitHub Discussions is now available for wappdriver. Ask Questions in the Discussions section Q/A category.

  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

I am excited to enable GitHub discussions for wappdriver.

**Comment by Ibonpython** [comment]
Hi
About Wappdriver, You have a good tool.

I see it is posible to have chrome driver in a no standard location, you can change the first time you use it. 
But, is It posible to do the same thing with a no standard location of chrome browser, for example if you use portable chrome browser?

Thanks a lot!!!

**Reply by aahnik** [reply]
Are you talking about https://portableapps.com/apps/internet/google_chrome_portable ?

i am not sure, as I have never used them. You need to experiment, and may be there is a tweak possible.
